docs/blog/2018-10-09-vscode-gatsby-development/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Using VS Code for Supercharged Gatsby.js Development"
+author: Michael Holtzman
+date: 2018-10-09
+tags: ["editor", "source", "documentation"]
+---
+
+The Gatsby.js platform and ecosystem is composed not only of the [core Inkteam members](https://www.gatsbyjs.com/about/), but a huge group of community members as well. I have no doubt that part of the success of Gatbsy is due to the core team really encouraging the community to be involved. They even give you [free swag for becoming a contributor](/docs/contributor-swag/).
+
+Now that I have convinced you to get involved, the first thing you should do is review the existing [how to contribute guide](/docs/how-to-contribute/). That will go over the basics of checking out the project and getting set up.
```

Comment:
First of all - thanks :)

But... I'd maybe just dive right into the topic at hand? People are going to come here for info on VSCode, so I'd recommend diving right in from the get go!

Reply:
no problem @DSchau.
Do you want me to just remove these two paragraphs totally and just get right into it?